backbone/src/main/res/values/strings.xml

```diff
@@ -32,74 +31,30 @@
     <string name="rsb_consent_section_more_info_withdrawing">Learn more about withdrawing</string>
 
     <!-- Consent sharing step -->
-    <string name="rsb_consent_share_title">Sharing Options</string>
-    <string name="rsb_consent_share_widely">Share my data with %1$s and qualified researchers
-        worldwide
-    </string>
-    <string name="rsb_consent_share_no">Not sharing your data</string>
-    <string name="rsb_consent_share_only">Only share my data with %1$s</string>
-    <string name="rsb_consent_share_description"><![CDATA[%1$s will receive your study data from your
-        participation in this study. \n\nSharing your coded study data more broadly (without
-        information such as your name) may benefit this and future research.<br/><br/><a href="%2$s">
-        Learn more about data sharing</a>]]>
-    </string>
 
     <!-- Consent doc step -->
-    <string name="rsb_consent_review_title">Review</string>
+    <string name="rsb_consent_review_title" tools:ignore="UnusedResources">Review</string>
```

Comment:
Why is this `tools:ignore="UnusedResources"` needed? this string is used in `ConsentTask`